src/renderer/scss/_gui.scss

```diff
@@ -1,4 +1,54 @@
-@import url(https://fonts.googleapis.com/css?family=Roboto:400,400i,500,500i,700);
+// Generic html styles used accross the App
+// component specific styling should go in the component scss file
+
+@font-face {
+  font-family: 'Metropolis';
+  font-weight: normal;
+  font-style: normal;
+  text-rendering: optimizeLegibility;
+  src: url('../../../static/font/metropolis/Metropolis-Medium.woff2') format('woff2');
+}
+
+@font-face {
+  font-family: 'Metropolis';
+  font-weight: 600;
+  font-style: normal;
+  text-rendering: optimizeLegibility;
+  src: url('../../../static/font/metropolis/Metropolis-SemiBold.woff2') format('woff2');
+}
+
+// @font-face {
```

Comment:
Still figuring out what font-weights we should use across the app.

Ignore for now